airrohr-firmware/airrohr-firmware.ino

```diff
@@ -604,22 +606,31 @@ const char data_first_part[] PROGMEM = "{\"software_version\": \"{v}\", \"sensor
 /*****************************************************************
  * Debug output                                                  *
  *****************************************************************/
-void debug_out(const String& text, const int level, const bool linebreak) {
+
```

Comment:
"if 0": Really?

Reply:
leftover. I only used this to measure the potential to spend some time on optimizing debug() statements. when removing all of them over 25kb of progmem is removed, so there is a very large potential. i've removed this now.